joomla/joomla-cms
# [4.0] Add default configuration to params in db on installation

Example for Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30027 but doesnt close it.

### Summary of Changes

Just one example of missing default params in the db when installing

### Testing Instructions

Install Joomla 4 - check the db and see the params are there
go to Joomla Global Config -> User Action Logs -> Save
Check the db #__extensions table for the com_actionlogs row, check the params are still there and still the same

### Actual result BEFORE applying this Pull Request

on installation there are missing params in the db

### Expected result AFTER applying this Pull Request

All defined params, for com_actionlogs, are set in the db at install time with sane defaults 

### Documentation Changes Required

none
